docs/concepts/custom-itemspanel.md

```diff
@@ -0,0 +1,71 @@
+---
+description: CONCEPTS
+---
+
+import exampleScreenshot from '/img/concepts/customitemspanel/ItemsControl-with-ItemPanel-and-Styles.png';
+
+# Custom ItemsPanel
+
+the following built-in controls, display their items in a StackPanel by default: 
+- [`ItemsControl`](../reference/controls/itemscontrol.md)
+- [`TreeView`](../reference/controls/detailed-reference/treeview-1.md)
+- [`Carousel`](../reference/controls/carousel.md)
+- [`Menu`](../reference/controls/menu.md)
+- [`ComboBox`](../reference/controls/combobox.md)
+- `ListBox` 
```

Comment:
Is there a reason ListBox specifically does not have a link?

Reply:
I could not find the page, happy to link to the right one, if you point it to me

Reply:
https://docs.avaloniaui.net/docs/reference/controls/listbox

Reply:
...I was blind, I'm not sure how I missed it. It's now updated and this on has also a VirtualizingStackPanel as default like the Carousel.